build.gradle

```diff
@@ -169,6 +169,12 @@ subprojects { subProject ->
 		testRuntime( libraries.mssql )
 		testRuntime( libraries.informix )
 
+		if (db.equalsIgnoreCase("sqlite")) {
+			dependencies {
+				testRuntime( libraries.sqlite )
```

Comment:
Why not providing it by default? The Oracle is a special case because it's not available on Maven Central.